samples/outbox/multi-tenant/Version_6/Receiver/Receiver.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Receiver</RootNamespace>
     <AssemblyName>Receiver</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
`AsyncLocal<T>` is 4.6. I think it is fair to require this since this is a very specific sample. Previous .NET editions did not have a good solution for async local state according to @danielmarbach